src/IotjsDebugger.ts

```diff
@@ -237,9 +237,12 @@ class IotjsDebugSession extends DebugSession {
   }
 
   protected restartRequest(response: DebugProtocol.RestartResponse, args: DebugProtocol.RestartArguments): void {
-    this.log('restartRequest: Not implemented yet', LOG_LEVEL.SESSION);
```

Comment:
A `this.log('restartRequest', LOG_LEVEL.SESSION);` message should be leave here.